kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/instance/ProcessInstanceUserTaskWithVariables.java

```diff
@@ -34,6 +34,9 @@
     @XmlElement(name = "name")
     private String name;
 
+    @XmlElement(name = "description")
+    private String description;
```

Comment:
Since this is applicable to both Processes and Cases, you should also do the same for `CaseUserTaskWithVariables` class

https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/model/cases/CaseUserTaskWithVariables.java